deepmd/pt/model/model/__init__.py

```
@@ -280,6 +283,9 @@ def get_standard_model(model_params):
         modelcls = DOSModel
     elif fitting_net["type"] in ["ener", "direct_force_ener"]:
         modelcls = EnergyModel
+        if "hessian_mode" in model_params.keys():
+            if model_params["hessian_mode"]:
```

Ruff

Use a single `if` statement instead of nested `if` statements

(SIM102)